### PR TITLE
[kube-prometheus-stack] Support multicluster dashboards only for etcd

### LINKED ISSUE #965: [kube-prometheus-stack] etcd Grafana dashboard breaks without "multicluster" enabled with other etcd metrics
**Describe the bug**
When `kube-prometheus-stack` is deployed e.g. together with `grafana/loki-stack` chart, with the following values:
```yaml
# For grafana/loki-stack
loki:
  serviceMonitor:
    enabled: true
promtail:
  serviceMonitor:
    enabled: true
```

`etcd` Grafana dashboard shows the wrong data, because etcd metrics coming from loki are use instead of ones from `kube-etcd` job.

Setting `.Values.grafana.sidecar.dashboards.multicluster` works, but it adds also fields for all other Kubernetes dashboards, which may not be desired.

There could be a separate option to enable multi-cluster only for etcd or just always enable it, to avoid this confusing scenario.

**Version of Helm and Kubernetes**:

Helm Version:

```console
$ helm version
version.BuildInfo{Version:"v3.5.3", GitCommit:"041ce5a2c17a58be0fcd5f5e16fb3e7e95fea622", GitTreeState:"dirty", GoVersion:"go1.15.8"}```
```
Kubernetes Version:

```console
$ kubectl version
Client Version: version.Info{Major:"1", Minor:"21", GitVersion:"v1.21.0", GitCommit:"cb303e613a121a29364f75cc67d3d580833a7479", GitTreeState:"clean", BuildDate:"2021-04-08T16:31:21Z", GoVersion:"go1.16.1", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"21", GitVersion:"v1.21.0", GitCommit:"cb303e613a121a29364f75cc67d3d580833a7479", GitTreeState:"clean", BuildDate:"2021-04-08T16:25:06Z", GoVersion:"go1.16.1", Compiler:"gc", Platform:"linux/amd64"}```
```

**Which chart**: prometheus-community/kube-prometheus-stack

**Which version of the chart**: 15.4.5

### PR DESCRIPTION
**Pull request by invidian**
#### What this PR does / why we need it:

This PR adds ability to only enable multicluster option for etcd
dashboard, so it doesn't have to be enabled for all Kubernetes
dashboards.

If one collects metrics from more than one etcd cluster on single
Kubernetes cluster, e.g. when having Loki deployed, which use etcd
internally, etcd dashboard in Grafana becomes useless without
multicluster option enabled.

This PR also syncs Grafana dashboards and does some styling improvements to kube-prometheus-stack values.yaml file.

#### Which issue this PR fixes
  - Closes #965

#### Notes for the reviewer

I tested it manually on my cluster and everything works as expected.

#### Checklist

- [x] [DCO](https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[prometheus-couchdb-exporter]`)

### CODE REVIEW

**Review by monotek — CHANGES_REQUESTED** [review]
Please merge master into your branch to get the "node not ready" problem in ci fixed.

**Comment by invidian** [comment]
>Please merge master into your branch to get the "node not ready" problem in ci fixed.

Done. CI is now passing.

File: charts/kube-prometheus-stack/templates/grafana/dashboards-1.14/prometheus.yaml [hunk 1]
```diff
@@ -573,6 +573,14 @@ data:
                         "stack": true,
                         "steppedLine": false,
                         "targets": [
+                            {
```
Comment (monotek) [inline charts/kube-prometheus-stack/templates/grafana/dashboards-1.14/prometheus.yaml:576]: https://github.com/prometheus-community/helm-charts/blob/e886d5ae0300be7fb51c0415d4ab4f2265301581/charts/kube-prometheus-stack/templates/grafana/dashboards-1.14/prometheus.yaml#L2-L4

**Review by monotek — CHANGES_REQUESTED** [review]

File: charts/kube-prometheus-stack/templates/grafana/dashboards-1.14/prometheus.yaml (hunk 1 above)
Comment (invidian) [reply charts/kube-prometheus-stack/templates/grafana/dashboards-1.14/prometheus.yaml:576]: :question: This change is pulled by the sync script.

**Comment by toboshii** [comment]
Not stale.

What needs to be done to get this merged?

**Comment by gkarthiks** [comment]
lgtm from the sync script, cc'ing @Xtigyro for additional review.

**Comment by invidian** [comment]
Let me rebase.

File: charts/kube-prometheus-stack/values.yaml [hunk 2]
```diff
@@ -679,6 +679,8 @@ grafana:
       ##
       annotations: {}
       multicluster: false
+      etcd:
+        multicluster: false
```
Comment (Xtigyro) [inline charts/kube-prometheus-stack/values.yaml:683]: Hey @invidian - could you please change this to be under a single `multicluster` and just have then `multicluster.global` and `multicluster.etcd`?

**Review by Xtigyro — CHANGES_REQUESTED** [review]

File: charts/kube-prometheus-stack/values.yaml (hunk 2 above)
Comment (invidian) [reply charts/kube-prometheus-stack/values.yaml:683]: Sure, should this be a breaking change then or should I do some hackery with `printf` to make it backwards compatible (as `multicluster` key type will change)?

File: charts/kube-prometheus-stack/templates/grafana/dashboards-1.14/prometheus.yaml (hunk 1 above)
Comment (invidian) [reply charts/kube-prometheus-stack/templates/grafana/dashboards-1.14/prometheus.yaml]: Ping @monotek

File: charts/kube-prometheus-stack/values.yaml (hunk 2 above)
Comment (Xtigyro) [reply charts/kube-prometheus-stack/values.yaml:683]: @Let's make it a breaking change without hackeries. The minor version of the chart will need to be bumped though.

Comment (invidian) [reply charts/kube-prometheus-stack/values.yaml]: Done. I bumped minor version, not major, so I didn't add any changelog to README.md.

**Review by gkarthiks — DISMISSED** [review]
lgtm

File: charts/kube-prometheus-stack/Chart.yaml [hunk 3]
```diff
@@ -18,7 +18,7 @@ name: kube-prometheus-stack
 sources:
   - https://github.com/prometheus-community/helm-charts
   - https://github.com/prometheus-operator/kube-prometheus
-version: 17.1.3
+version: 17.2.1
```
Comment (Xtigyro) [inline charts/kube-prometheus-stack/Chart.yaml:21]: @invidian This should be: `17.2.0`.

**Review by Xtigyro — CHANGES_REQUESTED** [review]

File: charts/kube-prometheus-stack/Chart.yaml (hunk 3 above)
Comment (invidian) [reply charts/kube-prometheus-stack/Chart.yaml]: I forgot the dummyness of charts release process :) Done dashboards syncing before adding a feature, without version bumping then.

**Review by Xtigyro — APPROVED** [review]
Thank ya, @invidian!

**Pull request merged**